cpp/ql/src/Likely Bugs/OO/UnsafeUseOfThis.ql

```diff
@@ -0,0 +1,211 @@
+/**
+ * @name Unsafe use of this in constructor
+ * @description A call to a pure virtual function using a this
```

Comment:
```suggestion
 * @description A call to a pure virtual function using a
```
Should this say "a pointer to an object"?

Reply:
I mean a `this` pointer in the object-oriented sense (as in the pointer `x` in a call like `x->f()`). Would it be more clear if we wrote `A call to a pure virtual function using a 'this' pointer [...]`?

Reply:
Got you! Yes, that would have avoided me reading it as a typo. Thanks

Reply:
Fixed in 4a7f9100e41d58cb7f8a96784307ce40fd2a4aee.